cmake/FindLuaJIT.cmake

```diff
@@ -15,35 +15,36 @@
 #
 #######################
 
-# FindLuaJIT.cmake
+# Findluajit.cmake
 #
 # This will define the following variables
 #
-#     LuaJIT_FOUND
+#     luajit_FOUND
 #
 # and the following imported targets
 #
-#     LuaJIT::LuaJIT
+#     luajit::luajit
 #
 
-# LuaJIT exports their own config since LuaJIT-1.5.0, but it isn't
-# present in the OpenSUSE libLuaJIT-devel-1.7.1 package and maybe others.
+# luajit exports their own config since luajit-1.5.0, but it isn't
+# present in the OpenSUSE libluajit-devel-1.7.1 package and maybe others.
 
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(LuaJIT luajit)
+pkg_check_modules(luajit luajit)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(LuaJIT
-    REQUIRED_VARS LuaJIT_LIBRARIES LuaJIT_INCLUDE_DIRS
+find_package_handle_standard_args(luajit
+    REQUIRED_VARS luajit_INCLUDE_DIRS luajit_LINK_LIBRARIES luajit_LIBRARIES luajit_LIBRARY_DIRS
+    HANDLE_COMPONENTS
```

Comment:
Does the LuaJIT from PkgConfig use any components?

Reply:
yes

Reply:
Probably good to have this then, thanks for adding it. 👍